rules/annotation.smk

```diff
@@ -46,15 +46,27 @@ rule vcfanno:
     wrapper:
         get_wrapper_path("vcfanno")
 
+rule pass:
+    input:
+       	"annotated/vcfanno/all.vep.vcfanno.vcf",
```

Comment:
minor thing. looks double tabbed?

Reply:
That is weird. I def see the spacing, but in the online text editor it looks fine and it's all spaces.

Check it out here:
https://github.com/ccmbioinfo/crg2/blob/730945093c6cb770d350b54d9f4c5e414664cb59/rules/annotation.smk